Documentation/gettingstarted/encryption.rst

```diff
@@ -184,7 +184,7 @@ To replace cilium-ipsec-keys secret with a new keys,
 
 .. code-block:: shell-session
 
-    KEYID=$(kubectl get secret -n kube-system cilium-ipsec-keys -o yaml|grep keys: | awk '{print $2}' | base64 -d | awk '{print $1}')
+    KEYID=$(kubectl get secret -n kube-system cilium-ipsec-keys -o yaml | grep -m 1 keys: | awk '{print $2}' | base64 -d | awk '{print $1}')
```

Comment:
Do we know for sure that the real key always comes first?

We're piping to `awk` anyway, so what about filtering like this:
```suggestion
    KEYID=$(kubectl get secret -n kube-system cilium-ipsec-keys -o yaml | awk '/^keys:/ {print $2}' | base64 -d | awk '{print $1}')
```

Reply:
I **think** it always comes first but I haven't officially confirmed it, so your suggestion is better. Thanks!